recipes/varys/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "0.1.0" %}
+
+package:
+  name: varys
+  version: {{ version }}
+
+source:
+  url: https://github.com/CLIMB-TRE/varys/archive/refs/tags/0.1.0.tar.gz
+  sha256: 792222af06b73c0b915c3c43474562d94cd7d8b72207a2937f1b2ed4a68acf25
+
+build:
+  number: 0
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+
+  run:
+    - python >=3.6
+    - pika >=1.3.0
+    - setuptools >=42
```

Comment:
I can see that `setuptools` is listed in the `setup.cfg` as an `install_requires`, but it's unclear that it's actually needed at runtime. It's fine to keep it here, but just a heads up since it's unusual. Usually it's only a run-time req if the package uses `pkg_resources`.

Reply:
Thanks for the feedback, have removed the req